bigtable-client-core/src/main/java/com/google/protobuf/BigtableZeroCopyByteString.java

```diff
@@ -33,19 +33,16 @@ public static ByteString wrap(final byte[] array, int offset, int length) {
     return new BoundedByteString(array, offset, length);
   }
 
-  // TODO:
-  // ZeroCopyLiteralByteString.wrap(this.buf, 0, this.count);
-
   /**
    * Extracts the byte array from the given {@link ByteString} without copy.
    * @param buf A buffer from which to extract the array.  This buffer must be
    * actually an instance of a {@code LiteralByteString}.
```

Comment:
toByteArray() will be called on any implementation other than LiteralByteString, included BoundedByteString.